Keas.Core/Domain/TeamTag.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System.ComponentModel.DataAnnotations;
+
+namespace Keas.Core.Domain
+{
+    public class TeamTag
```

Comment:
@jcubbage I think I'd like it much more if this was just called Tag.  I don't think there is going to be another class of tags w/o teams, and even if there was we can just make team optional in that case

Reply:
@srkirkland So, I started by calling it Tag, but that conflicted with the property of Tag. I couldn't come up with a good alternative other than TeamTag.Tag. Thoughts?